bool inquiry
---
algorithm_msgs/robot_state_base[] robot_base
